crates/cairo-profiler/src/trace_reader/sample.rs

```diff
@@ -82,19 +91,6 @@ impl Sample {
             );
         }
 
-        let syscall_counter_with_string: Vec<_> = resources
-            .syscall_counter
```

Comment:
We can remove this field from this struct in `cairo-annotations` now I believe. 

Before we remove this info from the trace data: would be nice to check manually if the amounts and kinds of syscalls you extracted from Sierra match the ones from our test trace data. Could you do that?

Reply:
I will handle the removal under https://github.com/software-mansion/cairo-annotations/issues/20 

> Before we remove this info from the trace data: would be nice to check manually if the amounts and kinds of syscalls you extracted from Sierra match the ones from our test trace data. Could you do that?

Yup, it matches. There is only one discrepancy, regarding "test" syscalls as they are not included in sierra, but I don't think they really matter that much in this context, as they are related to test execution only.

Reply:
Yea, cheatcodes (starknet test syscalls) are whatever, we shouldn't care about them